lib/client/kubesession.go

```diff
@@ -201,12 +197,47 @@ func (s *KubeSession) pipeInOut(stdout io.Writer, mode types.SessionParticipantM
 	go func() {
 		defer s.cancel()
 
-		handleNonPeerControls(mode, s.term, func() {
-			err := s.stream.ForceTerminate()
-			if err != nil {
-				fmt.Printf("\n\rError while sending force termination request: %v\n\r", err.Error())
+		if mode == types.SessionPeerMode {
+			// copy from the local input to the remote shell:
+			buf := make([]byte, 1024)
+
+			stdin := s.term.Stdin()
+			if s.term.IsAttached() {
+				stdin = escape.NewReader(stdin, s.term.Stderr(), func(err error) {
```

Comment:
Why are we using `escape.NewReader` in this case? Might be useful to clarify with a comment.